src/android/TTS.java

```diff
@@ -57,11 +65,12 @@ public void initialize(CordovaInterface cordova, final CordovaWebView webView) {
         tts.setOnUtteranceProgressListener(new UtteranceProgressListener() {
             @Override
             public void onStart(String s) {
-                // do nothing
+                Log.i(TAG, "onStart " + s);
             }
 
             @Override
             public void onDone(String callbackId) {
+                Log.i(TAG, "onDone " + callbackId);
```

Comment:
Should we remove the log.i statements now that its ready?

Reply:
Honestly, I think it could be nice to keep in the event we need to debug anything or make further changes, since it would only show up on a debug build in logcat, it's very safe.